src/Components/InstancesTable/helpers.js

```diff
@@ -1,11 +1,13 @@
-import { AWS_PROVIDER, AZURE_PROVIDER } from '../../constants';
+import { AWS_PROVIDER, AZURE_PROVIDER, GCP_PROVIDER } from '../../constants';
 
 export const SSHUsername = (provider) => {
   switch (provider) {
     case AWS_PROVIDER:
       return 'ec2-user';
     case AZURE_PROVIDER:
       return 'azureuser';
+    case GCP_PROVIDER:
+      return 'gcpuser';
```

Comment:
nit: this is needed for the follow-up PR, adding the instances table. but let's keep this